app/assets/javascripts/initializers/initNotifications.js

```diff
@@ -171,17 +170,25 @@ function initPagination() {
         .then(function(response) {
           if (response.status === 200) {
             response.text().then(function(html) {
-              const notificationsList = html.trim();
+              const markup = html.trim();
+
+              if (markup) {
+                const container = document.getElementById('articles-list');
+
+                const newNotifications = document.createElement('div');
+                newNotifications.innerHTML = markup;
+
+                paginator.remove();
+                container.append(newNotifications);
```

Comment:
Right now we're inserting the HTML coming from the server (which includes the paginator) in the previous paginator, basically nesting HTML in itself, which is pretty ugly.

With this we append the new notifications at the end of the list and remove the current paginator which is not needed after rendering.